app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -241,6 +254,14 @@ def index_organization
       return subscriptions
     end
 
+    def consumed_subscriptions(consumer, organization = current_organization)
+      entitlements = consumer.consumed_entitlements
+      entitlements.delete_if do |entitlement|
+        pool = Pool.find_pool(entitlement.poolId)
+        !pool.provider?(organization)
+      end
```

Comment:
The whole function, if I grep the code base, I don't see it being used -
consumed_subscriptions
On Sep 11, 2015 4:18 PM, "John Mitsch" <notifications@github.com> wrote:

> In app/controllers/katello/api/v2/subscriptions_controller.rb
> <https://github.com/Katello/katello/pull/5446#discussion_r39312436>:
>
> > @@ -241,6 +254,14 @@ def index_organization
> >        return subscriptions
> >      end
> >
> > +    def consumed_subscriptions(consumer, organization = current_organization)
> > +      entitlements = consumer.consumed_entitlements
> > +      entitlements.delete_if do |entitlement|
> > +        pool = Pool.find_pool(entitlement.poolId)
> > +        !pool.provider?(organization)
> > +      end
>
> the entitlement variable though? .where(:id => entitlement.poolId)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Katello/katello/pull/5446/files#r39312436>.
>